configs/dataset/ndf/_default.yaml

```diff
@@ -1,5 +1,9 @@
 
 # This is the general structure of the datamodule for NDF tasks.
+defaults:
+  - ../@train_dset.demo_dset.occlusion_cfg: _augmentations
```

Comment:
lol this is unholy. love it.

Reply:
yeah probably a better way to do it, but it works :)